Lib/inspect.py

```diff
@@ -1638,7 +1638,29 @@ def getclosurevars(func):
 
 # -------------------------------------------------- stack frame extraction
 
-Traceback = namedtuple('Traceback', 'filename lineno function code_context index')
+_Traceback = namedtuple('_Traceback', 'filename lineno function code_context index')
+
+class Traceback(_Traceback):
+    def __new__(cls, filename, lineno, function, code_context, index, positions=None):
```

Comment:
FrameInfo's positions is keyword only, Traceback's isn't. Be consistent on that. (no opinion which is better - users won't typically be constructing these themselves)